pccommon/tests/config/test_render_config.py

```diff
@@ -5,21 +5,21 @@
 
 multi_assets = DefaultRenderConfig(
     assets=["data1", "data2"],
-    render_params={"colormap_name": "terrain", "rescale": [-1000, 4000]},
+    render_params={"colormap_name": "terrain", "rescale": "-1000,4000"},
     minzoom=8,
 )
 
 single_asset = DefaultRenderConfig(
     assets=["data1"],
-    render_params={"colormap_name": "terrain", "rescale": [-1000, 4000]},
+    render_params={"colormap_name": "terrain", "rescale": "-1000,4000"},
```

Comment:
Just checking it's intentional that some `rescale` params are using string values and some lists of one string. As discussed, they're treated identically, but may be good to be consistent here.

Reply:
Sorry, not intentinal.